cmd/apiserver/app/server/run_server.go

```diff
@@ -56,7 +57,7 @@ func RunServer(opts *ServiceCatalogServerOptions, stopCh <-chan struct{}) error
 
 func runEtcdServer(opts *ServiceCatalogServerOptions, stopCh <-chan struct{}) error {
 	etcdOpts := opts.EtcdOptions
-	glog.V(4).Infoln("Preparing to run API server")
+	glog.Infof("Preparing to run Service Catalog API server %s (built %s)", version.Get().String(), version.Get().BuildDate)
```

Comment:
can we put this in the caller of runEtcdServer? I do not want there to end up duplicating code when we get a CRD server.